src/fetchers.js

```diff
@@ -122,24 +230,21 @@ function fetchers(params = {}) {
   const dirindex = (params['content.index'] || 'index.html,README.html').split(',');
   const infos = getPathInfos(params.path || '/', params.rootPath || '', dirindex);
 
-  const staticOpts = {
+  const staticOpts = resolveOpts({
```

Comment:
add check to avoid double fetching if content and static repo are the same.

Reply:
Done in 
0a21d29